crates/kitsune_p2p/dht/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "kitsune_p2p_dht"
+version = "0.0.1"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
you need to actually fill out the Cargo.toml - docs, readme, license, etc... in order for it to be publish-able - just copy stuff out of the kitsune_p2p one.

Reply:
I will get to that after any other feedback